### ISSUE TITLE
Can we start a list of all the working and not working maps?

### ISSUE THREAD

**Issue by WilliamCSalt** [InvoxiPlayGames/BO3MacFix#21]
Some that I tested

Working:
Shoot Haus

Not Working:
Vae Victis
Der riese Declassified :'(

**Comment by adamwawrzynkowski** [comment]
Working:

- MAZESCAPE
- HALLE DER TOTEN
- THE IRON DRAGON
- RAINY DEATH
- KYASSURU
- Minecraft Shi No Numa
- DAYBREAK
- SHREK ZOMBIES
- CALL OF THE DEAD
- Mob of the Dead Remastered
- NIGHTMARE
- LEVIATHAN
- TOWN REIMAGINED

**Comment by Exzh34** [comment]
> Working:
> 
> * MAZESCAPE
> * HALLE DER TOTEN
> * THE IRON DRAGON
> * RAINY DEATH
> * KYASSURU
> * Minecraft Shi No Numa
> * DAYBREAK
> * SHREK ZOMBIES
> * CALL OF THE DEAD
> * Mob of the Dead Remastered
> * NIGHTMARE
> * LEVIATHAN
> * TOWN REIMAGINED
> * Die Rise Remastered
> * Mob of The Dead Remastered (sean. remaster)

**Comment by kylejericson** [comment]
I can't get Call of the Dead to work I get stuck at the loading screen forever. @Exzh34

**Comment by theantachable** [comment]
I'm on an M1 Macbook Pro 2021

played the following perfectly with my friends (they're on windows)
- Kowloon
- Eingesperrt
- Rust 10.0
- Der Graben

Not working:
- Campingplatz Der Untoten

**Comment by Exzh34** [comment]
> I can't get Call of the Dead to work I get stuck at the loading screen forever. @Exzh34

me too, I just added the maps I tried to the list

**Comment by vsynth21** [comment]
So far I have

Not working:
Call of the Dead
Der Riese: Declassified
Karma Rezort

Working but stuttery: 
Town Reimagined

Working:
Der Riese (by gcp345)
Kowloon
Mob of the Dead (by copforthat)
Town Remastered
Truck Stop

**Comment by InvoxiPlayGames** [comment]
Thanks for all the info and testing! I'm going through each of the broken and stuttery ones and seeing what I can do to fix the problems in v1.1.

**Comment by Svetrikk** [comment]
> Working:
> 
> 
> 
> - MAZESCAPE
> 
> - HALLE DER TOTEN
> 
> - THE IRON DRAGON
> 
> - RAINY DEATH
> 
> - KYASSURU
> 
> - Minecraft Shi No Numa
> 
> - DAYBREAK
> 
> - SHREK ZOMBIES
> 
> - CALL OF THE DEAD
> 
> - Mob of the Dead Remastered
> 
> - NIGHTMARE
> 
> - LEVIATHAN
> 
> - TOWN REIMAGINED
> 
> 

Daybreak isn't working for me. It loads the intro cutscene (which has artifacts like its corrupted or something) and then it stays on black screen. I get this error in the console: Error: We have not pumped socketrouter for 25,sec

**Comment by MileZer0** [comment]
Daybreak also doing the same for me. Tried playing DEAD HIGH as well and it kept giving me error message about WRITE location, no such file, i think it has something to do w the fact that it wants to save progress on that map. Not sure how to fix

**Comment by InvoxiPlayGames** [comment]
Could you re-try some of these maps with the latest v1.2-beta1 release? https://github.com/InvoxiPlayGames/BO3MacFix/releases

**Comment by vsynth21** [comment]
> Could you re-try some of these maps with the latest v1.2-beta1 release? https://github.com/InvoxiPlayGames/BO3MacFix/releases

Absolutely! I love this mod and what it's already capable of, and I really appreciate all the work you've put in to make custom zombies possible on machines we never thought possible. I'll keep editing this list as I go along

Compatibility list for v1.2beta1

**Don't Work (Stuck at loading screen):**
Call of the Dead (by copforthat)
Der Riese: Declassified
Karma Rezort
My Actual Cabin
Vae Victis

Working but too stuttery (Mostly due to recursive errors from calling stacks with GSCs with missing line information):
Nuketown Zombies
Karma Courtyard
Nova Genesis
Town Reimagined

Works:
Blizzard Outpost
Club Penguin Zombies
Europa
Facility Remastered (By DWILS)
Daybreak
Der Riese (by gcp345)
Dunder Mifflin
Farm
Gestort
Halle Der Toten
Kalter Tod
Kowloon
La Lumiere
Les Cendres Du Mal
Lost Apartments
Maison
Maison EXT
Metal Gear Solid (Ran at ~30fps)
Mob of the Dead (by copforthat) (Random stuttering has been fixed. Some stuttering during plane steps & afterlife entry/reentry, but overall very playable)
Nightmare (UI error displays start of match, game plays fine after)
Outbreak Reimagined
Overlook Remastered (By DWILS)
Return to Bus Depot
Return to Das Herrenhaus
Shisha no Teikou
The Reichstag
Truck Stop
Town Remastered
Water Park Zombies (Ran at ~40fps)
Water Park Zombies: After Hours (Ran at ~40fps)

**Comment by rom7nn** [comment]
Interesting that people’s working and not working maps are different. Daybreak loads a glitchy loading screen for 0-2 seconds then dies. On the other hand Call of the Dead has been working for me.

**Comment by rx-dev** [comment]
To add my own experiences:
- Town Remastered is fairly laggy / fps drops even on low
- Shoothause plays fine, stutters here and there
- Iron Dragon is superb, plays better than some vanilla maps except when the Panzer spawns and looks at me, it goes to shit
- Call of the Dead dies at the loading screen
- Tunnel (return to tranzit) is laggy similar to town remastered
- Die Rise Remastered worked mostly ok, lag spike here and there

Macbook Pro 14in 2023, M2Pro
